packages/graphback-codegen-resolvers/src/formatters/ApolloTsResolverFormatter.ts

```diff
@@ -46,13 +46,13 @@ const generateTypeResolvers = (context: TargetResolverContext, name: string, opt
   let resolverType = '';
   let typedImports = ''
   if (options.types) {
```

Comment:
We do not have this done for our own CLI so this code can be tested with graphql cli. 
We can add that later.